staging/src/k8s.io/cri-client/pkg/logs/logs_test.go

```diff
@@ -289,7 +289,8 @@ func TestReadRotatedLog(t *testing.T) {
 	err = file.Close()
 	assert.NoErrorf(t, err, "could not close file.")
 
-	time.Sleep(20 * time.Millisecond)
+	// waitLog.wait may wait 1 second
+	time.Sleep(1500 * time.Millisecond)
```

Comment:
should we test both - with a second wait and without 1 second wait? So both codepath will be tested?